com.reprezen.swagedit.openapi3/src/com/reprezen/swagedit/openapi3/assist/ext/ResponseCodeContentAssistExt.java

```diff
@@ -0,0 +1,85 @@
+package com.reprezen.swagedit.openapi3.assist.ext;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.google.common.base.Predicate;
+import com.google.common.base.Strings;
+import com.google.common.collect.Iterators;
+import com.google.common.collect.Lists;
+import com.reprezen.swagedit.core.assist.Proposal;
+import com.reprezen.swagedit.core.assist.ext.ContentAssistExt;
+import com.reprezen.swagedit.core.model.AbstractNode;
+import com.reprezen.swagedit.core.schema.TypeDefinition;
+import com.reprezen.swagedit.openapi3.Activator;
+
+public class ResponseCodeContentAssistExt implements ContentAssistExt {
+
+    private static final JsonPointer pointer = JsonPointer.compile("/definitions/responses");
+
+    private final List<String> baseCodes = Lists.newArrayList("200", "300", "400", "500");
+    private final ArrayNode statusCodes;
+
+    public ResponseCodeContentAssistExt() {
+        statusCodes = init();
+    }
+
+    private ArrayNode init() {
+        try {
+            final URL file = Activator.getDefault().getBundle().getResource("resources/status-codes.json");
+            try {
+                return (ArrayNode) new ObjectMapper().readTree(file);
+            } catch (IOException e) {
+                return null;
+            }
+        } catch (NullPointerException e) {
+            // For standalone tests, bundle cannot be loaded.
+            return null;
+        }
+    }
+
+    @Override
+    public boolean canProvideContentAssist(TypeDefinition type) {
+        return type != null && pointer.equals(type.getPointer());
+    }
+
+    @Override
+    public Collection<Proposal> getProposals(TypeDefinition type, AbstractNode node, String prefix) {
+        Collection<Proposal> proposals = Lists.newArrayList();
+
+        for (Iterator<JsonNode> it = statusCodes(prefix); it.hasNext();) {
+            JsonNode current = it.next();
+            String code = current.get("code").asText();
+            String description = current.get("description").asText();
+            String phase = current.get("phrase").asText();
+
+            proposals.add(new Proposal(code, code, description, phase));
```

Comment:
Can we keep `displayString` as it is now and add a colon(`:`) to the end of the `replacementString` so user don't have to type it. For example:
* `displayString` = `200`
* `replacementString` = `200:`